lib.nix

```diff
@@ -64,99 +64,60 @@ rec
   mkMetadataKey = name: version:
     "checksum ${name} ${version} (registry+https://github.com/rust-lang/crates.io-index)";
 
-  # a record:
-  #   { "." = # '.' is the directory of the cargotoml
-  #     [
-  #       {
-  #         name = "rand";
-  #         url = "https://github.com/...";
-  #         checkout = "/nix/store/checkout"
-  #       }
-  #     ]
+  # Gets all git dependencies in Cargo.lock as a list.
+  # [
+  #   {
+  #     name = "rand";
+  #     url = "https://github.com/...";
+  #     checkout = "/nix/store/checkout"
   #   }
+  # ]
   findGitDependencies =
-    { cargotomls
```

Comment:
Is there any functionality that is being lost by not looking into Cargo.toml?

Reply:
As far as I've checked, we don't lose anything; I've been using this patched version myself for a couple of past months and I haven't found any issues either.